packages/database/src/migrations/20210804234015-AddKoBoDataSourceEntities-modifies-data.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  const schools = (
+    await db.runSql(`
+      SELECT code, TRIM(code, 'LA_sch_') AS kobo_id
+      FROM entity
+      WHERE code LIKE 'LA_sch_%'
+    `)
+  ).rows;
+
+  return schools.map(row =>
+    insertObject(db, 'data_service_entity', {
+      id: generateId(),
+      entity_code: row.code,
+      config: {
+        kobo_id: row.kobo_id,
+      },
```

Comment:
Wow fully expected `data_service_entity` to specify which service the mapping was for, but I guess with different config entries we could have one row that mapped multiple services. Might need to use more specific service codes than `dhis_id` and `kobo_id` in future, but we can leave that for another day